test/websocket-server.test.js

```diff
@@ -137,6 +137,32 @@ describe('WebSocketServer', function () {
     });
   });
 
+  describe('#address', function () {
+    it('returns the address of the server', function (done) {
+      const wss = new WebSocket.Server({ port: 0 }, () => {
+        const addr = wss.address();
+        assert.deepStrictEqual(wss._server.address(), addr);
+        wss.close(done);
+      });
+    });
+
+    it('throws an error when operating in "noServer" mode', function () {
+      const wss = new WebSocket.Server({ noServer: true }, () => {
+        assert.throws(() => {
+          wss.address();
+        }, /This server is operating in "noServer" mode/);
```

Comment:
Please use `/^Error: This server is operating in "noServer" mode$/` so we also validate the type of the error.